.github/workflows/validate-pull-request.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Add Jira Issue Code to Pull Request
-        if: github.event.pull_request != ''
+        if: github.event.pull_request == opened
```

Comment:
Shouldn't these be strings?

Reply:
after checking, I think it should be github.event.pull_request.opened== true

Reply:
It seems, yes, though check https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Workflows%20and%20actions%20have%20access%20to%20a%20huge%20amount